packages/dappmanager/src/modules/hostServices/runServices.ts

```diff
@@ -0,0 +1,28 @@
+import path from "path";
+import fs from "fs";
+import params from "../../params";
+import { shellHost } from "../../utils/shell";
+
+/**
+ * Service runners. Helps ensure no typos
+ */
+type ServiceName = "update-docker-engine.service";
+
+/**
+ * Run a service for the hostService folder
+ * @param serviceName "update-docker-engine.service"
+ * sytemd service info: https://www.freedesktop.org/software/systemd/man/systemd.service.html
+ */
+export async function runService(
+  serviceName: ServiceName,
+  args = ""
+): Promise<string> {
+  const servicePath = path.resolve(
+    params.HOST_SERVICES_SOURCE_DIR,
+    serviceName
+  );
+  if (!fs.existsSync(servicePath))
+    throw Error(`Host service ${serviceName} not found`);
+
+  return await shellHost(`systemctl start ${serviceName} ${args}`);
```

Comment:
Keep in mind that if *.service file is ever edited, before this command you need to run `systemctl daemon-reload`. I don't know whether that will be ever the case, but keep it in mind.

Reply:
god point, I will edit the PR taking this into account

Reply:
For this specific service, `systemctl enable update-docker-engine.service` is not executed since the behaviour desired is to be executed only on demmand and not on boot, so `systemctl daemon-reload` would not be necessary

Anyway might be useful for any other service there might be added in the future so I will add it